Extension/src/LanguageServer/protocolFilter.ts

```diff
@@ -23,10 +23,12 @@ export function createProtocolFilter(me: Client, clients: ClientCollection): Mid
         didOpen: (document, sendMessage) => {
             if (clients.checkOwnership(me, document)) {
                 me.TrackedDocuments.add(document);
-                me.provideCustomConfiguration(document).then(() => {
-                    sendMessage(document);
-                }, () => {
-                    sendMessage(document);
+                me.notifyWhenReady(() => {
```

Comment:
I think you should await `requestWhenReady` on the QTUS request inside `provideCustomConfiguration` in client.ts (line 643) instead of using `notifyWhenReady` here.